string name
geometry_msgs/Vector3 position
geometry_msgs/Quaternion orientation
geometry_msgs/Twist velocity
float32 pitch
float32 yaw
